src/components/VmDetails/style.css

```diff
@@ -101,3 +101,9 @@
 div.base-card.cell-card {
   flex-basis: auto;
 }
+
+@media only screen and (max-width: 600px) {
```

Comment:
Is this `@media` section relevant for this PR or is it a holdover from another?

Reply:
(same as in #1028)

Reply:
These are because I branched this branch out of 1028. I'll re-branch and these shouldn't be an issue

Reply:
Looks like your branch for this PR is still tracking your branch for #1028...

Reply:
Done